cli/src/utils.js

```diff
@@ -79,6 +79,14 @@ export function getServices(dir) {
         .map(s => services[s].image);
 }
 
+export async function getMissingImgs(dir) {
+    const docker = new Docker({});
+    const availableImgs = await docker.command(`images --format "{{.Repository}}:{{.Tag}}"`).then(
```

Comment:
can you use await twice insteand of `then`?

Reply:
✅